charts/umbrella/Chart.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   - condition: portal.enabled
     name: portal
     repository: https://eclipse-tractusx.github.io/charts/dev
-    version: 1.8.1
+    version: 2.0.0
```

Comment:
Could you please restrict your changes to bpdm?
I know it won't work as a whole as long as portal isn't upgrade as well  (that's why we work on a separate upgrade branch until all components are ready). I'd upgrade portal, centraldip and sharedidp all together in a separate PR.

Reply:
Understood. I'll revert portal version upgrade. I'm ready to help with the portal, centraldip, and sharedidp upgrades if needed.

Reply:
Sure! In that case, should I create a separate pull request for each component, or do I upgrade them in one?

Reply:
I think one pull request with the upgrade of all 3 components would be also good :)